Texture problem in Polyhedron Demo

The items using textures in the Polyhedron Demo generate a glsl error : 
```
QOpenGLShader::compile(Fragment): 0(11) : error C7532: global function texture requires "#version 130" or later
```
The texturing code will need to be rewritten.

--- a/Polyhedron/demo/Polyhedron/resources/shader_with_texture.f
+++ b/Polyhedron/demo/Polyhedron/resources/shader_with_texture.f
@@ -5,5 +5,5 @@ uniform sampler2D s_texture;
  
 void main(void) 
 { 
-  gl_FragColor = vec4(vec3(texture(s_texture, f_texCoord))*fColors, 1.0);
+  gl_FragColor = vec4(vec3(texture2D(s_texture, f_texCoord))*fColors, 1.0);
 }  

--- a/Polyhedron/demo/Polyhedron/resources/shader_with_textured_edges.f
+++ b/Polyhedron/demo/Polyhedron/resources/shader_with_textured_edges.f
@@ -5,5 +5,5 @@ uniform highp sampler2D s_texture;
  
 void main(void) 
 { 
-  gl_FragColor = vec4(vec3(texture(s_texture, f_texCoord))*fColors, 1.0); 
+  gl_FragColor = vec4(vec3(texture2D(s_texture, f_texCoord))*fColors, 1.0); 
 }

